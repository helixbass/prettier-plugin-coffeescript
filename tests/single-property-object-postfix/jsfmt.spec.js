run_spec(__dirname, ['coffeescript'], {respectBreak: ['object']})
run_spec(__dirname, ['coffeescript'], {respectBreak: []})
