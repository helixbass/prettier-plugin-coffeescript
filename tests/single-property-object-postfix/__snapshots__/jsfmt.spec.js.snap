// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test.coffee 1`] = `
a: b if c

BREAK

a:
  b if c

BREAK

a: b while c

BREAK

a:
  b while c

BREAK

a: b for c in d

BREAK

a:
  b for c in d
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{ a: b } if c

BREAK

a:
  b if c

BREAK

a: b while c

BREAK

a:
  b while c

BREAK

{ a: b } for c in d

BREAK

a:
  b for c in d

`;

exports[`test.coffee 2`] = `
a: b if c

BREAK

a:
  b if c

BREAK

a: b while c

BREAK

a:
  b while c

BREAK

a: b for c in d

BREAK

a:
  b for c in d
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{ a: b } if c

BREAK

a: (b if c)

BREAK

a: b while c

BREAK

a: (b while c)

BREAK

{ a: b } for c in d

BREAK

a: (b for c in d)

`;
